addons/assigngear/Cfg3DEN.hpp

```diff
@@ -196,6 +178,112 @@ class Cfg3DEN
                 };
             };
         };
+        class GVAR(AmmoBox): TitleWide {
+            onLoad = QUOTE( [ARR_2('onLoad', _this)] call FUNC(gui_vehicleGear_selector) );
+            attributeLoad = QUOTE([_value] call FUNC(gui_vehicleGear_load));
+            attributeSave = QUOTE([] call FUNC(gui_vehicleGear_save));
+            h = (22 * ATTRIBUTE_CONTENT_H + 1) * GRID_H;
+            class Controls: Controls
+            {
+                class CategoryTitle: ctrlStatic {
+                    x = 0;
+                    y = 0;
+                    w = ATTRIBUTE_TITLE_W * GRID_W;
+                    h = SIZE_XL * GRID_H;
+                    text = "Category"
+                    style = 0x01;
```

Comment:
Might want to use the macros provided in `\3DEN\UI\resincl.inc`

Reply:
Resincl contains only ID's so im not sure what you mean and i already use the provided EDEN macros so i'm not sure what you are on about.

Reply:
It also contains macros such as 
`#define ST_RIGHT 0x01`
`#define ST_VERTICAL 0x01`
`#define CT_BUTTON 1`